DigitalOffice/Tests/FileServiceTests/Commands/AddNewFileCommandTests.cs

```diff
@@ -0,0 +1,109 @@
+﻿using FluentValidation;
+using Moq;
+using NUnit.Framework;
+using System;
+using LT.DigitalOffice.FileService.Database.Entities;
+using LT.DigitalOffice.FileService.RestRequests;
+using LT.DigitalOffice.FileService.Commands;
+using LT.DigitalOffice.FileService.Commands.Interfaces;
+using LT.DigitalOffice.FileService.Repositories.Interfaces;
+using LT.DigitalOffice.FileService.Mappers.Interfaces;
+
+namespace FileServiceUnitTests.Commands
+{
+    public class AddNewFileCommandTests
+    {
+        private DbFile newFile;
+        private FileCreateRequest fileRequest;
+        private IAddNewFileCommand command;
+        private Mock<IFileRepository> repositoryMock;
+        private Mock<IValidator<FileCreateRequest>> validatorMock;
+        private Mock<IMapper<FileCreateRequest, DbFile>> mapperMock;
+
+        [SetUp]
+        public void Initialization()
+        {
+            repositoryMock = new Mock<IFileRepository>();
+            validatorMock = new Mock<IValidator<FileCreateRequest>>();
+            mapperMock = new Mock<IMapper<FileCreateRequest, DbFile>>();
+
+            newFile = new DbFile
+            {
+                Id = Guid.NewGuid(),
+                Content = Convert.FromBase64String("RGlnaXRhbCBPZmA5Y2U="),
+                Extension = ".txt",
+                IsActive = true,
+                Name = "DigitalOfficeTestFile",
+                AddedOn = DateTime.UtcNow
+            };
+
+            fileRequest = new FileCreateRequest
+            {
+                Content = "RGlnaXRhbCBPZmA5Y2U=",
+                Extension = ".txt",
+                Name = "DigitalOfficeTestFile"
+            };
+
+            mapperMock
+                .Setup(f => f.Map(It.IsAny<FileCreateRequest>()))
+                .Returns(newFile);
+            command = new AddNewFileCommand(repositoryMock.Object, validatorMock.Object, mapperMock.Object);
+        }
+
+        [Test]
+        public void SuccessfulAddNewFileTest()
```

Comment:
Hm... We stopped adding a _Test_ ending for test methods

Reply:
At the end of the test name should be Test (convention).